### DISCUSSION TITLE
Turn off Admin Area Button

### DISCUSSION THREAD

**Discussion by Setman590** [e107inc/e107#5150] [Q&A Core] [answered]
Hi Guys,

I was wondering, when I am not logged in I have a Go to Admin Area blue button that shows up in the middle on my website I am working on. Is there a way to get rid of it? I already have a sign in area on the right side of my screen.

Thanks in advance

**Comment by Alex-e107nl** [comment, marked as answer]
Hi,
go to admin - content - Welcome Messages - edit your welcome mesage, drop the button... 
(i think)

**Comment by Setman590** [comment]
That did it. Thank you
